### PR TITLE
Adds Kangtai protocol (Commonly used in Cotech devices from Clas Ohlson)

### PR DESCRIPTION
**Pull request by k0d**

### CODE REVIEW

File: src/devices/kangtai.c [hunk 1]
```diff
@@ -0,0 +1,77 @@
+#include "rtl_433.h"
```
Comment (merbanan) [inline src/devices/kangtai.c]: Add a proper copyright header and preferably a tiny description of the protocol. Look in other files how it is done.

File: src/devices/kangtai.c [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+
+static int kangtai_callback(bitbuffer_t *bitbuffer) {
+	uint8_t *bytes = bitbuffer->bb[0];
+	data_t *data;
+	uint8_t k0, k1, k2, k3, k4, k5;
+	uint8_t g0, g1, g2, g3, g4, g5;
+	char table1[] = { 1, 8, 4, 14, 2, 7, 13, 6, 15, 12, 0, 10, 3, 11, 5, 9 };
+	char table2[] = { 15, 6, 0, 11, 5, 2, 10, 4, 12, 13, 14, 8, 1, 9, 3, 7 };
+	uint16_t address;
+
```
Comment (merbanan) [inline src/devices/kangtai.c]: Add timestamp code here. Look at the template decoder for an example.

File: src/devices/kangtai.c [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+		g3 = table2[k3] ^ k2;
+		g2 = table2[k2] ^ k1;
+		g1 = table2[k1] ^ k0;
+		g0 = table2[k0];
+	}
+
+	bytes[0] = (g5 << 4) | g4;
+	bytes[1] = (g3 << 4) | g2;
+	bytes[2] = (g1 << 4) | g0;
+
```
Comment (merbanan) [inline src/devices/kangtai.c]: Can you describe what this transformation does ? It is not obvious by looking at the code.

**Comment by merbanan** [comment]
The code looks good. Fix my comments and send some sample files and the patch is ready for merging.

**Comment by k0d** [comment]
I've made the changes as you requested. I'll add some sample files to the tests project once I figure out how to capture them :)

**Comment by merbanan** [comment]
Look good. Merged.

**Comment by k0d** [comment]
Is it possible for you to remove my 2 commits from this repository,
apparently the protocol should not be released publicly.

**Comment by Victoare** [comment]
I know it's been a while, but what happened with this?
I cannot find in the current master.

**Pull request merged**